modules/users/client/components/UserSummary.js

```diff
@@ -0,0 +1,43 @@
+// External dependencies
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'styled-components';
+
+// Internal dependencies
+import Avatar from '@/modules/users/client/components/Avatar.component';
+
+export default function UserSummary({ user }) {
+  const UserSummary = styled.div.attrs({
```

Comment:
[Try to define the styled component outside the component it's used in.](https://www.styled-components.com/docs/faqs#why-should-i-avoid-declaring-styled-components-in-the-render-method)